### ISSUE TITLE
when will this library support Cloudlfare Queue wrapper?

### ISSUE THREAD

**Issue by ad6025b** [cloudflare/workers-honeycomb-logger#47]
the queue handler fails to work when it is wrapped by the fetch handler..

**Comment by evanderkoogh** [comment]
Yeah, the queue handler isn't supported in this library. As you asked in the other issue, this library is going to be obsolete pretty soon with the otel library maturing. So I don't foresee any new features being added in this library.

As I am no longer at Cloudflare, I can also no longer commit to this repo.
